src/script/components/Conversation/Conversation.tsx

```diff
@@ -374,6 +382,27 @@ const ConversationList: FC<ConversationListProps> = ({
             isRightSidebarOpen={isRightSidebarOpen}
           />
 
+          {activeCalls.map(call => {
+            const conversation = conversationState.findConversation(call.conversationId);
+            const callingViewModel = mainViewModel.calling;
+            const callingRepository = callingViewModel.callingRepository;
+            return (
+              conversation &&
+              smBreakpoint && (
+                <div className="calling-cell" key={conversation.id}>
+                  <CallingCell
+                    classifiedDomains={undefined}
+                    call={call}
+                    callActions={callingViewModel.callActions}
+                    callingRepository={callingRepository}
+                    conversation={conversation}
+                    multitasking={callingViewModel.multitasking}
+                  />
+                </div>
+              )
+            );
```

Comment:
It will be more readable ;)

```suggestion
            if (!conversation && !smBreakpoint) {
                return null;
            }
            
            return (
                <div className="calling-cell" key={conversation.id}>
                  <CallingCell
                    classifiedDomains={undefined}
                    call={call}
                    callActions={callingViewModel.callActions}
                    callingRepository={callingRepository}
                    conversation={conversation}
                    multitasking={callingViewModel.multitasking}
                  />
                </div>
            );
```

Reply:
You mean `or` @przemvs? 😄 
```js
if (!conversation || !smBreakpoint) {
     return null;
}
```

Reply:
He have in the code && :D

Reply:
yeah, but to keep the logic you need to use `||` instead of `&&` when using negation

Reply:
we want both values to be truthy to display calling cell fragment right?

Reply:
It's or indeed, I corrected it